### ISSUE TITLE
KA10: RFNM from IMP

### ISSUE THREAD

**Issue by larsbrinkhoff** [rcornwell/sims#107]
ITS expects to get RFNM messages from the IMP.  Or really, from the remote host.  But for our purposes, the IMP simulation would generate these messages.

The original IMP would normally only accept five outstanding messages to some particular host without getting acknowledged by an RFNM.  ITS keeps track of this count and stops sending messages when the IMP wold refuse them.

**Comment by rcornwell** [comment]
I was not seeing any problems during my testing. Also looking at the source did not see it actually tracking the connections.

**Comment by larsbrinkhoff** [comment]
No, you wouldn't have seen it, because the tracking was lost due to bit rot.

The 1985 file SYSTEM; INET 115 had this tracking in place.  At some point, the old IMP code was saved as IMPOLD WTHNCP.  Later the code in INET was moved to the new IMP file as the routine IMPCTS, but IMPOLD was not updated.

So now that I have copied IMPCTS from the new IMP to IMPOLD, it will track RFNM messages again.
